scrollingelement.js

```diff
@@ -28,9 +28,27 @@ if (!('scrollingElement' in document)) {
 			return isFrameset ? null : body;
 		};
 
-		Object.defineProperty(document, 'scrollingElement', {
-			'get': scrollingElement
-		});
+		if ('defineProperty' in Object) {
+			Object.defineProperty(document, 'scrollingElement', {
+				'get': scrollingElement
+			});
+		} else if ('__defineGetter__' in document) {
+			// Firefox < 4, Safari < 5, Opera < 12
```

Comment:
Hmm, Opera 11.64 supports `Object.defineProperty`, so I guess `Opera < 12` should be removed here.